extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -319,9 +342,68 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
   }
 
   /** @override */
+  buildCallback() {
+    super.buildCallback();
+
+    this.autoFormat_ =
+        this.element.getAttribute('data-auto-format') || '';
+
+    if (this.isResponsive()) {
+      // Attempt to resize to the correct height.
+      const viewport = this.getViewport();
+      return this.attemptChangeSize(
+          AmpAdNetworkAdsenseImpl.getResponsiveHeightForContext_(
+              viewport.getSize()),
+          undefined);
```

Comment:
I feel like we need to set the viewport width here as well b/c we want to "fix" it before rendering, right? E.g. if the user resizes (re-orients) we can't really change width anymore anyway, right?